extensions/amp-analytics/0.1/variables.js

```diff
@@ -211,6 +211,17 @@ export class VariableService {
     this.register_('LINKER_PARAM', (name, id) =>
       this.linkerReader_.get(name, id)
     );
+
+    this.register_('VIDEO_STATE', (id, property) => {
```

Comment:
I'm not very familiar with VIDEO_STATE variable, should we include it in video component instead of analytics?

Reply:
It looks like it was added in [this pr](https://github.com/ampproject/amphtml/pull/10820) as a part of the [Video Analytics feature](https://github.com/ampproject/amphtml/issues/9902).

I don't see any clues as to why its a global url-replacement versus amp-analytics specific. Judging from the I2I though, I feel like it should be fine to move it. What are your thoughts?

Reply:
/cc @ampproject/wg-ui-and-a11y

Reply:
Maybe the getter here can be exported/exposed from the `video-manager`, but so far there's not a clear boundary in how video analytics features are split.

I'd be okay with adding this, but I don't find it necessary for now:

```js
class VideoManager {
  // ...
  getAnalyticsStateVariable(videoId, property) {}
}
```

Reply:
As discussed offline, we can't move the getter into `amp-analytics`, but we can move it onto the video service:

```js
class VideoManager {
  // move implementation of current analytics variable getter here:
  getAnalyticsState(videoId, property) {}
}

class UrlReplacements {
  // then you still decrease core bundle size when implemented here like:
  this.setAsync('VIDEO_STATE', (id, property) =>
    Services.videoManagerForDoc(this.ampdoc)
      .getAnalyticsState(id, property)
  )
}
```

(the video service is bundled with video extensions, not with core).